scout/server/blueprints/alignviewers/controllers.py

```diff
@@ -23,32 +24,56 @@ def make_sashimi_tracks(institute_id, case_name, variant_id, build="38"):
     Returns:
         display_obj(dict): A display object containing case name, list of genes, lucus and tracks
     """
+    build = "38"  # This feature is only available for RNA tracks in build 38
 
-    # Collect locus coordinates. Take into account that variant can hit multiple genes
     variant_obj = store.variant(document_id=variant_id)
-    variant_genes_ids = [gene["hgnc_id"] for gene in variant_obj.get("genes", [])]
+    _, case_obj = institute_and_case(store, institute_id, case_name)
+
     # Initialize locus coordinates it with variant coordinates so it won't crash if variant gene(s) no longer exist in database
-    locus_start_coords = [variant_obj.get("position")]
-    locus_end_coords = [variant_obj.get("end")]
-    gene_symbols = []
+    locus_start_coords = []
+    locus_end_coords = []
+
+    # Check if variant coordinates are in genome build 38
+    # Otherwise do variant coords liftover
+    if build not in str(case_obj.get("genome_build")):
+        client = EnsemblRestApiClient()
+        mapped_coords = client.liftover(
```

Comment:
🥇Really nice! I suppose it might hang or get slow, but that could be said for many things on the align viewer pages.

Reply:
The web service it generally quite fast, I use it for doing the liftover for matchmaker matches and haven't had problems so far..